internal/test/helpers/webhook.go

```diff
@@ -83,7 +83,7 @@ func initializeWebhookInEnvironment() {
 		klog.Fatalf("Failed to get information for current file from runtime")
 	}
 	root := path.Join(path.Dir(filename), "..", "..", "..")
-	configyamlFile, err := os.ReadFile(filepath.Clean(filepath.Join(root, "config", "webhook", "manifests.yaml")))
+	configyamlFile, err := os.ReadFile(filepath.Clean(filepath.Join(root, "config", "govmomi", "webhook", "manifests.yaml")))
```

Comment:
Do we also need to load 
```
	configyamlFile, err := os.ReadFile(filepath.Clean(filepath.Join(root, "config", "supervisor", "webhook", "manifests.yaml")))
```

So supervisor hooks are also added to the testenv?

Reply:
Just did a quick check, seems supervisor testenv is started [here](https://github.com/kubernetes-sigs/cluster-api-provider-vsphere/blob/main/controllers/vmware/test/controllers_suite_test.go#L69)

Reply:
@zhanggbj Sounds reasonable